titus-server-master/src/main/java/com/netflix/titus/master/jobmanager/service/DefaultV3JobOperations.java

```diff
@@ -390,6 +390,9 @@ public Completable recordTaskPlacement(String taskId, Function<Task, Task> chang
 
     @Override
     public Observable<Void> moveServiceTask(String sourceJobId, String targetJobId, String taskId) {
+        if(!jobManagerConfiguration.enableMoveTaskApi()){
```

Comment:
To follow the JavaBean/Archaius naming convention, it should be `isMoveTaskApiEnabled`.